lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -30,7 +30,7 @@ module Billing #:nodoc:
     class OrbitalGateway < Gateway
       include Empty
 
-      API_VERSION = '7.1'
+      API_VERSION = '7.7'
```

Comment:
Upgrading from 7.1 to 7.7 seems like a big jump in version to me. Any concerns here? Should the upgrade be separate from the credentials update?

Reply:
I don't think so, shouldn't be a big deal if there was no change in behavior with tests.